coderd/database/types.go

```diff
@@ -35,6 +35,40 @@ type NotificationsSettings struct {
 	NotifierPaused bool      `db:"notifier_paused" json:"notifier_paused"`
 }
 
+type InboxNotificationReadStatus string
```

Comment:
@dannykopping this is a new part of the PR that you'd probably want to check , related to our discussion and the new version of SQL queries.